mne/viz/ica.py

```diff
@@ -160,6 +161,12 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
     hist_ax = var_ax_divider.append_axes("right", size="33%", pad="2.5%")
     var_ax.scatter(range(len(epoch_var)), epoch_var, alpha=0.5,
                    facecolor=[0, 0, 0], lw=0)
```

Comment:
The original scatter should have corrected x axis when we remove some segments (so that for example the x should be [0, 2, 3] if we rejected epoch index 1).

Reply:
ignore this too, `epoch_var` has all segments, and overplotting some points later with red shouldn't be a problem

Reply:
Oh yes, because I reconstruct it before sending it to the plot_ica_properties helper. I had a doubt for a sec ^^'